.github/workflows/add-depr-ticket-to-depr-board.yml

```diff
@@ -60,13 +60,13 @@ jobs:
           gh api graphql -f query='
             query($org: String!, $number: Int!) {
               organization(login: $org){
-                projectNext(number: $number) {
+                projectV2(number: $number) {
                   id
```

Comment:
why do we use this extra code in the tcril-engineering workflow? https://github.com/openedx/tcril-engineering/blob/main/.github/workflows/add-GHrequest-to-team-board.yml#L58-L62

Reply:
I noticed that as I was transferring those changes here: I think they are left over from setting the status to ToDo.  We needed the ids of the status there originally, but no longer do.